leakpro/attacks/utils/shadow_model_handler.py

```diff
@@ -107,7 +107,7 @@ def _filter(self:Self, data_size:int, online:bool)->list[int]:
     def create_shadow_models(
         self:Self,
         num_models:int,
-        shadow_population: np.ndarray,
```

Comment:
Nothing really that needs a change but just out of curiosity, why a list over a np.array?